packages/gatsby/src/utils/stack-trace-utils.js

```diff
@@ -13,7 +17,9 @@ const getNonGatsbyCallSite = () =>
       callSite =>
         callSite &&
         callSite.getFileName() &&
-        !callSite.getFileName().includes(gatsbyLocation)
+        !callSite.getFileName().includes(gatsbyLocation) &&
```

Comment:
Wonder if there are any others we want to add in here?

Reply:
I'm not exactly happy with those (it was kind of trial and error, seeing what will pop up there) and we should only handle packages that are there because of gatsby internals (and nothing else) - so I don't think we should add any more (right now)

Reply:
Okay, that's fair